frontend/src/components/BreadCrumbsWrapper.tsx

```diff
@@ -0,0 +1,78 @@
+'use client'
+
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Breadcrumbs, BreadcrumbItem as HeroUIBreadcrumbItem } from '@heroui/react'
+import upperFirst from 'lodash/upperFirst'
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+const ROUTES_WITH_PAGE_LAYOUT = [
```

Comment:
I also think this route detection logic is somewhat duplicated: regex patterns in here vs string matching in useBreadcrumbs 🤔